src/topp/LuciphorAdapter.cpp

```diff
@@ -597,7 +598,22 @@ class LuciphorAdapter :
     f.getOptions() = options;
     f.load(in, exp);
     exp.sortSpectra(true);
-    
+
+    // check if mz/intenisty precision have different values - set them to 64-bit
```

Comment:
here you just state what you do, it would be better if you explain why this is necessary (e.g. bug in Luciphore and link to OpenMS issue / Luciphore issue). This would also help us to determine if / when we can remove this condition again